mod_wtjshoppingfavorites/tmpl/bootstrap5-badge-icon-only-for-mobile.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * WT JoomShopping Favorites is an alternative wish list (favorite products) for JoomShopping based on coockies.
+ * @package     WT JoomShopping Favorite
+ * @version     2.0.1
+ * @Author      Sergey Tolkachyov, https://web-tolk.ru
+ * @copyright   Copyright (C) 2020 Sergey Tolkachyov
+ * @license     GNU/GPL 3.0
+ * @since       1.0.0
+ * @link        https://web-tolk.ru/en/dev/joomshopping/wt-joomshopping-favorite.html
+ */
+defined('_JEXEC') or die('Restricted access');
+
+use Joomla\CMS\Router\Route;
+
+if (!empty($product_ids) && !is_null($product_ids) && count($product_ids) > 0)
```

Comment:
По идее этого достаточно. 

```suggestion
if (!empty($product_ids) )
```